### DISCUSSION TITLE
Paid membership website for Csla support

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#4261] [Ideas]
Hi fellow Csla developers,
We all have bills to pay, so I suggest Rocky (@rockfordlhotka ) start a paid membership website that provides Csla support.

Kind Regards

**Comment by StefanOssendorf** [comment]
He sort of has? https://lhotka.net/

**Reply by rockfordlhotka** [reply]
I certainly do provide paid mentoring, training, and consulting around CSLA.

**Reply by Chicagoan2016** [reply]
@rockfordlhotka I was thinking of monthly or yearly subscription. It will be easier for us to ask our employers to pay for it.

**Reply by rockfordlhotka** [reply]
@Chicagoan2016  I do offer a retainer style model that might work for you. Email me to discuss - rocky at marimer.llc

**Reply by Chicagoan2016** [reply]
@rockfordlhotka My current employer is not very enthusiastic about paying for it but I can pay out of my pocket : ).

**Reply by Chicagoan2016** [reply]
@rockfordlhotka Good morning, I just sent you an email.
Kind Regards

**Reply by MarkOverstreet** [reply]
Why not just support CSLA here by doing a monthly donation?
